frontend/packages/react-app/src/components/organisms/UUIDDistributionTargets/index.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import { Box, Typography, Card, Button, TextField } from "@material-ui/core";
+import { DISTRIBUTOR_ACTIONS } from "../../../reducers/distributorForm";
+import { UUID_ACTIONS, UUIDState } from "../../../reducers/uuid";
+
+export interface TargetsProps {
+  readonly uuidState: UUIDState;
+  readonly uuidDispatch: React.Dispatch<UUID_ACTIONS>;
+  readonly distributorFormDispatch: React.Dispatch<DISTRIBUTOR_ACTIONS>;
+}
+
+// TODO add input form
+const UUIDDistributionTargets: React.FC<TargetsProps> = ({
+  uuidState,
+  uuidDispatch,
+  distributorFormDispatch,
+}) => (
+  <>
+    <Card>
+      <Box p={4}>
+        <Box mt={4}>
+          <Typography variant={"h4"}>
+            1. Input distribution target quantity
+          </Typography>
+        </Box>
+        <>
+          <Box my={4} style={{ textAlign: "center" }}>
+            <Typography>Distribution target quantity</Typography>
+          </Box>
```

Comment:
IMO: I thought that "number of unique URLs" would be easier to understand than "target quantity".